File: client/iih_stub/iih.ci_stub/iih.ci.ord/srvref/d/MedicalWfTpAbbrEnum.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.ComponentModel;

namespace iih.ci.ord.srvref.d
{
	public class MedicalWfTpAbbrEnum {
        [Description("门诊")]
		public const String OP="OP";//门诊
        [Description("急诊")]
		public const String ER="ER";//急诊
        [Description("住院")]
		public const String IP="IP";//住院
        [Description("体检")]
		public const String PE="PE";//体检
        [Description("家床")]
		public const String FM="FM";//家床
   }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/srvref/d/MedicalWfTpAbbrEnum.java
package iih.ci.ord.srvref.d;

import xap.sys.appfw.orm.desc.ent.dataobject.DmEnumDesc;

public interface MedicalWfTpAbbrEnum {

    @DmEnumDesc(name="门诊",description="门诊英文简称")
    public static final String OP="OP";
    @DmEnumDesc(name="急诊",description="急诊英文简称")
    public static final String ER="ER";
    @DmEnumDesc(name="住院",description="住院英文简称")
    public static final String IP="IP";
    @DmEnumDesc(name="体检",description="体检英文简称")
    public static final String PE="PE";
    @DmEnumDesc(name="家床",description="家庭病床英文简称")
    public static final String FM="FM";
}	
